QgisModelBaker/gui/export.py

```diff
@@ -359,17 +395,7 @@ def on_process_finished(self, exit_code, result):
             self.buttonBox.addButton(QDialogButtonBox.Close)
         else:
             if self.export_without_validate():
-
-                # button is removed to define order in GUI
-                for button in self.buttonBox.buttons():
-                    if button.text() == self.export_button_name:
-                        self.buttonBox.removeButton(button)
-                # Check if button was previously added
-                self.remove_export_without_validate_button()
-
-                self.buttonBox.addButton(self.export_without_validate_button_name,
-                                         QDialogButtonBox.AcceptRole).setStyleSheet("color: #aa2222;")
-                self.buttonBox.addButton(self.export_button_name, QDialogButtonBox.AcceptRole)
+                self.set_export_without_validation()
```

Comment:
The two methods `export_without_validate()` and `set_export_without_validate()` look like getters and setters for a property. But if I'm not wrong, one is to change the default behavior of a button and the other one to trigger the export (?)
Would it make sense to rename to `set_default_export_button_action(action: ButtonActionEnum)` or similar?

Reply:
You are right. Agreeing on the input.